### DISCUSSION TITLE
What do you think we should/could test?

### DISCUSSION THREAD

**Discussion by JulieBB23** [gc-da11yn/Content-and-ux#8] [Ideas]
It has been now almost 2 months that the site is live, up and running.
We will do a review of the current state and will identify which area we should focus.
As part of user experience, research and design there are a lot that we could cover, but where should we start?

Here what i was thinking we could do use some user testing and/or research:

- Is the description for each category are right?
- Should we review the structure of the content?
- Is there any page that would benefice from a visual redesign?

I would like to here from you :0) share with us what you like to research and explore to expand our user experience and to adapt our website.

**Comment by JulieBB23** [comment]
@emmatseng-ux  and @Syracopoulos  i think you have some ideas?

**Reply by Syracopoulos** [reply]
Hi! Was any end user testing completed prior to the launch? I'm wondering if we can take a step back and review the site hierarchy and usability. I'm receiving some feedback that it is difficult to navigate - so looking to have some data to respond with :)

**Reply by emmatseng-ux** [reply]
@Syracopoulos Hey Ellen! Yes my colleague and I did some follow-up testing from the initial survey that the accessibility/DAT team ran. We conducted 2 rounds of treejack testing in EN and FR with the AWG group+GC colleagues. It has been over a year since so my memory is a bit foggy, I’ll pull up our documentation and share them with you later this afternoon if you'd like? :)

**Reply by Syracopoulos** [reply]
Excellent. I am new to the UX/UI world but I am very eager to learn! If you don't mind sharing and then could I contact you if I have other questions?

**Reply by emmatseng-ux** [reply]
@Syracopoulos Absolutely! I’ll round up those pieces for you. We can also schedule an initial call and I can walk you through what the team has done so far. If you have  any additional questions, you can reach out to me at any time :)

**Comment by JulieBB23** [comment]
Something that we should look at is how we should format long page such as "[Best practices fir accessible virtual events](https://github.com/gc-da11yn/user-experience-planning/discussions/8)"

**Comment by JulieBB23** [comment]
I want to review and see which format we should us, specifically on the homepage latest update : [4.7 Dates](https://design.canada.ca/style-guide/#wp4-7:~:text=rule%20for%20numbers-,4.7%20Dates,-Write%20dates%20without)
